src/python/pants/backend/project_info/tasks/export_dep_as_jar.py

```diff
@@ -252,67 +257,95 @@ def initialize_graph_info(self, targets_map):
 
     return graph_info
 
-  def generate_targets_map(self, all_targets, runtime_classpath):
+  def _get_all_targets(self, targets):
+    additional_java_targets = []
+    for t in targets:
+      if isinstance(t, ScalaLibrary):
+        additional_java_targets.extend(t.java_sources)
+    targets.extend(additional_java_targets)
+    return targets
+
+  def _get_targets_to_print(self, all_targets, target_roots_set):
+    def target_is_between_roots(t):
+      transitive_dependencies = set(
+        DependencyContext.global_instance().dependencies_respecting_strict_deps(t) if hasattr(t, 'strict_deps') else \
+          DependencyContext.global_instance().all_dependencies(t)
+      )
+      return len(transitive_dependencies.intersection(target_roots_set)) > 0
+
+    def target_is_a_resource(t):
+      return isinstance(t, Resources)
+
+    printable_targets = []
```

Comment:
Just a thought: to match better with intellij concepts, how do you feel about renaming to `modulable_targets`?

Reply:
I prefer `modulizable` because `modulable` means "can be modulated", which is not what we want in this case.
But I like the suggestion, thanks!